infrastructure/bin/post_deployment.sh

```diff
@@ -48,7 +48,9 @@ if
     mkdir --parents /tmp/bootstrap/cache /tmp/api/storage/framework/cache/data && \
     chown www-data:www-data /tmp/bootstrap/cache && \
     chown -R www-data:www-data /tmp/api/storage && \
```

Comment:
Do we still need these lines if we are going to append it with optimize?

Reply:
I'm actually not sure. We can test it to see?

Reply:
Okay, removed to see if we can go without.

[remove possible redundant permission change](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12372/commits/14f8e53a6980fb4295f05543114a4c1496d2399f)

Reply:
still works fine on Dev 👍 .  Good to go then!